packetbeat/main.go

```diff
@@ -5,6 +5,17 @@ import (
 
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/packetbeat/beater"
+
+	// import support protocol modules
+	_ "github.com/elastic/beats/packetbeat/protos/amqp"
```

Comment:
By automate I mean the generation of this list based on the directories that exist. In metricbeat I put the list in a different package: https://github.com/elastic/beats/tree/master/metricbeat/include

The advantage of this if someone creates a beat with only one module he can either include all other modules and metricsets he wants one by one or he can just link to `_ "github.com/elastic/beats/metricbeat/include" and gets all the official modules of metricbeat without having to update it all the time.